src/binary-search-tree.js

```diff
@@ -5,6 +5,81 @@ class BinarySearchTree {
     this.left = null;
     this.right = null;
   }
+<<<<<<< HEAD
```

Comment:
Don't forget to resolve these conflicts. You don't want to just leave these in your code.